clients/pkg/promtail/server/server.go

```diff
@@ -51,6 +54,8 @@ type Config struct {
 	ExternalURL       string `yaml:"external_url"`
 	HealthCheckTarget *bool  `yaml:"health_check_target"`
 	Disable           bool   `yaml:"disable"`
+	Reload            bool   `yaml:"reload"`
+	NewByReload       bool
```

Comment:
What is this for?

Reply:
This is redundant, invalid variables left over from my development process